src/stylesheets/common/_buttons.scss

```diff
@@ -22,12 +22,17 @@ input[type="search"] + .btn {
 
 // Dark purple button
 .btn-mapzen {
-  background-color: nth($mz-darkpurples, 1);
-  color: #ffffff;
+  @extend .btn-primary;
+}
+
+.btn-primary {
+  background-color: $mz-darkpurple-1;
```

Comment:
Are`$mz-darkpurple-1` and `$mz-darkpurple-2` defined anywhere?  We're using `nth($mz-darkpurples, X)` everywhere else.